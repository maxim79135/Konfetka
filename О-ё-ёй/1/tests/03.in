20
66049 28387 43817 -61596 71762 52498 26318 6347 64131 -25573 -77963 -3520 -48509 -14485 -33188 -32722 18414 -48849 -14760 -6600 
