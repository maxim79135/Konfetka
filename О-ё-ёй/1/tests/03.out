-77963 -61596 -48849 -48509 -33188 -32722 -25573 -14760 -14485 -6600 -3520 6347 18414 26318 28387 43817 52498 64131 66049 71762 